xclim/indicators/atmos/_synoptic.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""Synoptic indicator definitions."""
+
+from xclim import indices
+from xclim.core.indicator import Daily
+
+__all__ = ["jetstream_metric_woolings"]
+
+
+class JetStream(Daily):
+    """Indicator involving daily u- and/or v-component wind series."""
+
+
+jetstream_metric_woolings = JetStream(
+    identifier="jetstream_metric_woolings",
+    units="days",
+    standard_name="daily_latitude_and_strength_of_jetstream",
+    long_name="Daily latitude and strength of maximum smoothed zonal wind speed",
+    description="{freq} daily latitude and strength of maximum smoothed zonal wind speed",
+    cell_methods="",
+    compute=indices.jetstream_metric_woolings,
+)
```

Comment:
```suggestion
jetstream_metric_woolings = JetStream(
    identifier="jetstream_metric_woolings",
    units=["degrees_North", "m s-1"],
    long_name=["Latitude of maximum smoothed zonal wind speed", "Maximum strength of smoothed zonal wind speed"],
    description=["Daily latitude of maximum smoothed zonal wind speed", "Daily maximum strength of smoothed zonal wind speed"],
    compute=indices.jetstream_metric_woolings,
)
```
As the "compute" function (the indice) returns 2 outputs (DataArrays), we must give metadata fields for each input. Passing list is one way, another would have been to pass a list of dictionaries like `cf_attrs=[{'units': ...}, {'units': ....}]`.

I removed the standard name as it was not from the official list. (http://cfconventions.org/Data/cf-standard-names/current/build/cf-standard-name-table.html)

The cell methods could theoretically be generated following the instructions in the [CF conventions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#cell-methods), but xclim lacks a mechanism to inject the correct dimension names... Like, to be super exact we would want to have something like : "lon: mean pressure: mean lat: maximum", but with the correct dimension names instead of "lon", "lat" and "pressure". As we can't, I prefer removing "cell_methods" entirely.